.gitlab-ci.yml

```diff
@@ -140,12 +141,12 @@ build-rust-doc-release:            &build
     expire_in:                     7 days
     paths:
     - ./crate-docs
-  <<:                              *build-only
+  # <<:                              *build-only
   tags:
     - linux-docker
-  before_script:
-    - sccache -s
-    - ./scripts/build.sh
+  # before_script:
+  #   - sccache -s
+  #   - ./scripts/build.sh
```

Comment:
these can probably be remove entirely, don't remember why it was put there.

Reply:
I included my experiment in this branch accidentially, sorry, it should't be there.
This job won't work without building WASM binaries.

Reply:
also hadn't tested it before.
```
error: couldn't read node/executor/src/../../runtime/wasm/target/wasm32-unknown-unknown/release/node_runtime.compact.wasm: No such file or directory (os error 2)
  --> node/executor/src/lib.rs:26:100
   |
26 | native_executor_instance!(pub Executor, node_runtime::api::dispatch, node_runtime::native_version, include_bytes!("../../runtime/wasm/target/wasm32-unknown-unknown/release/node_runtime.compact.wasm"));
   |                                                                                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

error: Could not compile `node-executor`.
```
but doesn't work without it.